connect/utils.go

```diff
@@ -0,0 +1,15 @@
+package connect
```

Comment:
Can we put these in main.go instead? We already have some logging stuff there. That way we avoid clogging the directory with too many files.

Reply:
moved